src/Options.cpp

```diff
@@ -90,6 +83,10 @@ void Options::ResetToDefaults()
     PatchDisableVignette = false;
     PatchDisableBoundaryTeleport = false;
     PatchDisableWin7Vsync = false;
+
+    RemoveDeadBindings = true;
+    EnableImGuiAssertions = true;
```

Comment:
Still not a fan of this being enabled by default, I don't see why users should receive assertion on things they can't repair, it's just going to generate massive support request for something we can't help with